src/osd/modules/input/input_retro.cpp

```diff
@@ -970,83 +956,71 @@ class joystick_input_retro : public retroinput_module
 			if (devinfo == nullptr)
 				continue;
 
-	      		// add the axes
-
+			// add the axes
 			devinfo->device()->add_item(
-				"LX",
+				"LSX",
 				static_cast<input_item_id>(ITEM_ID_XAXIS),
 				generic_axis_get_state<std::int32_t>,
 				&joystate[i].a1[0]);
 			devinfo->device()->add_item(
-				"LY",
+				"LSY",
 				static_cast<input_item_id>(ITEM_ID_YAXIS),
 				generic_axis_get_state<std::int32_t>,
 				&joystate[i].a1[1]);
 
 			devinfo->device()->add_item(
-				"RX",
+				"RSX",
 				static_cast<input_item_id>(ITEM_ID_RXAXIS),
 				generic_axis_get_state<std::int32_t>,
 				&joystate[i].a2[0]);
 			devinfo->device()->add_item(
-				"RY",
+				"RSY",
 				static_cast<input_item_id>(ITEM_ID_RYAXIS),
 				generic_axis_get_state<std::int32_t>,
 				&joystate[i].a2[1]);
-            
-         devinfo->device()->add_item(
+
+			devinfo->device()->add_item(
 				"L2",
-				static_cast<input_item_id>(ITEM_ID_RZAXIS),
+				static_cast<input_item_id>(ITEM_ID_ZAXIS),
```

Comment:
why are you changing the analog triggers? the default is good here testing in Out Run: R2 accelerates, L2 breaks.

Reply:
My mistake. I thought RZAXIS meant right. Just checked other input driver and you are right.